test/components/component_test.rb

```diff
@@ -110,8 +110,8 @@ def test_all_components_pass_through_classes
   def test_all_components_support_inline_styles
     default_args = { style: "width: 100%;" }
     COMPONENTS_WITH_ARGS.each do |component, args, proc|
-      render_component(component, default_args.merge(args), proc)
-      assert_selector("[style='width: 100%;']", visible: :all)
+      node = render_component(component, default_args.merge(args), proc)
+      assert_equal(true, node.children.attribute("style")&.value&.include?("width: 100%"))
```

Comment:
`assert_selector` looks for an exact match which we won't see with `Primer::Alpha::Tooltip` so I updated this assertion.